docker-compose.yml

```
@@ -36,8 +36,8 @@ x-sentry-defaults: &sentry_defaults
       <<: *depends_on-healthy
     kafka:
       <<: *depends_on-healthy
-    postgres:
-      <<: *depends_on-healthy
+    pgcat:
+      <<: *depends_on-default
```

Let's make this depends on healthy instead.

```suggestion
      <<: *depends_on-healthy
```